src/main/com/cqfn/degitx/simulator/Request.kt

```diff
@@ -0,0 +1,15 @@
+package com.cqfn.degitx.simulator
+
+/**
+ * Stub for Request sent via Network to Address
+ */
+class Request {
+    constructor(size: Int) {
```

Comment:
@IlyaMoskva we must know sender address of the request

Reply:
@g4s8 , You're right. Added.